boards/arm/nrf52833_pca10100/doc/index.rst

```diff
@@ -0,0 +1,223 @@
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth,           |
+|           |            | ieee802154           |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| USB       | on-chip    | usb                  |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See `nRF52833 PDK website`_ and `Nordic Semiconductor Documentation library`_
+for a complete list of nRF52833 Development Kit board hardware features.
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED1 (green) = P0.13
+* LED2 (green) = P0.14
+* LED3 (green) = P0.15
+* LED4 (green) = P0.16
+
+Push buttons
+------------
+
+* BUTTON1 = SW1 = P0.11
+* BUTTON2 = SW2 = P0.12
+* BUTTON3 = SW3 = P0.24
+* BUTTON4 = SW4 = P0.25
+* BOOT = SW5 = boot/reset
+
+Programming and Debugging
+*************************
+
+Applications for the ``nrf52833_pca10100`` board configuration can be
+built and flashed in the usual way (see :ref:`build_an_application`
+and :ref:`application_run` for more details); however, the standard
+debugging targets are not currently available.
+
+Flashing
+========
+
+Follow the instructions in the :ref:`nordic_segger` page to install
+and configure all the necessary software. Further information can be
+found in :ref:`nordic_segger_flashing`. Then build and flash
+applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Here is an example for the :ref:`hello_world` application.
+
+First, run your favorite terminal program to listen for output.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the board nRF52 DK
+can be found. For example, under Linux, :code:`/dev/ttyACM0`.
+
+Then build and flash the application in the usual way.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: nrf52833_pca10100
+   :goals: build flash
+
+Debugging
+=========
+
+Refer to the :ref:`nordic_segger` page to learn about debugging Nordic boards with a
+Segger IC.
+
+
+Testing the LEDs and buttons in the nRF52833 PDK
+************************************************
+
+There are 2 samples that allow you to test that the buttons (switches) and LEDs on
+the board are working properly with Zephyr:
+
+.. code-block:: console
```

Comment:
I wasn't going to comment on this, but since there's another edit to make, it's better to link to the sample documentation here, so replace this code-block with a list:
```
* :ref:`blinky-sample`
* :ref:`button-sample`
```

Reply:
Added that too!